calculations/do_calculations.py

```diff
@@ -0,0 +1,57 @@
+def temp_given_month(data):
```

Comment:
Quite a vague name, doesn't make sense what the function wants to achieve. Names should be meaningful and descriptive.

Reply:
Avoid using name like `data`, `list` etc. Your parameter names are as important as any other.